src/components/MaterialUI/Surfaces/ExpansionPanel.js

```diff
@@ -12,7 +12,7 @@ const useStyles = makeStyles(theme => ({
 	expansionPanelHeader: {},
 }));
 
-const SectionExpansionPanel = ({
+const ExpansionPanel = ({
```

Comment:
Was there any SectionExpansionPanel tests that need to be changed?

Reply:
everything was already changed